source/beamline/AM3DCoordinatedSystemControl.h

```diff
@@ -100,6 +100,22 @@ class AM3DCoordinatedSystemControl : public AMPseudoMotorControl
 	  * Adds the provided control to the list of child controls.
 	  */
 	virtual void addChildControl(AMControl *control);
+
+	/*!
+	  * Virtual function which performs the calculation required to transform a
+	  * vector movement from the global system to the arbitrary transformed one.
+	  * \param globalVector ~ A vector in the global coordinate system which will
+	  * be tranformed into the arbitrary transformed system.
+	  */
+	virtual QVector3D globalAxisToPrime(const QVector3D& globalVector) const {return globalVector;}
```

Comment:
I will look at this again, as it seemed to to me as I moved them just now. I know my thinking at the time was that if a coordinate system control like this doesn't specify a means of transforming, then if it is asked to transform a vector it should just return that vector untransformed. 
I will look further whether this should actually be here, or at a level below (after all a coordinated system doesn't necessarily have to have a concept of being transformed).